### PR TITLE
Remove ok comments from InputRequireThisTryWithResources

### PR DESCRIPTION
**Pull request by smitsekhadiaa**
Solves issue #13213
Removed ok comments from InputRequireThisTryWithResources

### CODE REVIEW

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -164,52 +164,6 @@
             files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThisReceiver.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThisStatic.java"/>
-  <suppress id="UnnecessaryOkComment"
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:167]: The changes lgtm....

**Review by MANISH-K-07 — COMMENTED** [review]
Please change your commit message to follow checkstyle rules : 

```
[ERROR] Failures: 
[ERROR]   CommitValidationTest.testCommitMessageHasProperStructure:216 Commit 962146672f9348a4eb8498f798c239b473db1407 message: "Remove ok comments from InputRequireThisTryWithResources\n" is invalid
Proper commit message should adhere to the following rules:
    1) Must match one of the following patterns:
        ^Issue #\d+: .*$
        ^Pull #\d+: .*$
        ^(minor|config|infra|doc|spelling|dependency|supplemental): .*$
    2) It contains only one line of text
    3) Must not end with a period, space, or tab
    4) Commit message should be less than or equal to 200 characters

The rule broken was 1
```

Also, update your PR description to address issue #13213

**Comment by smitsekhadiaa** [comment]
Changes done @MANISH-K-07 
Thankyou

**Comment by romani** [comment]
Please fix CI https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=19880&view=logs&j=c902ebb4-c9f8-5f09-4e17-ff78fbbc842e&t=9ca98c81-ff64-58f0-9d03-a23ac1c4a111&l=1330

Before push validate on on your local by `mvn clean verify `

**Comment by nrmancuso** [comment]
@smitsekhadiaa do you plan to continue this PR?

**Comment by romani** [comment]
Already done by somebody else

**Pull request closed without merging**